spec/gocd/pipeline_status/pipeline_spec.rb

```diff
@@ -33,6 +38,11 @@
       @raw_pipeline = {'name' => 'pipeline 1', 'activity' => 'sleeping', 'lastBuildStatus' => 'Failure'}
     end
 
+    it '#status should be failure' do
+      @pipeline = GOCD::Pipeline.new @raw_pipeline
```

Comment:
I know that all the other specs are using `@pipeline` instance variable, its my bad. For the new tests could you please use a local variable instead of instance variable.

Reply:
I have changed this but the changes are not yet being reflected in this PR. I guess this is because of the [problems Github are currently having](https://status.github.com/messages) so it should get updated soon.

Reply:
I have changed this but the changes are not yet being reflected in this PR. I guess this is because of the [problems Github are currently having](https://status.github.com/messages) so it should get updated soon.